_components/figure/figure.md

```diff
@@ -1,8 +1,21 @@
 ---
+component:
+  status: alpha
```

Comment:
Just want to confirm this status prevents it from rendering in the menu, right?

Reply:
Yes, for now they need to have a `status: ready` to appear